src/management/index.js

```diff
@@ -90,6 +90,7 @@ class ManagementClient {
    * @param   {boolean} [options.retry.enabled=true]                Enabled or Disable Retry Policy functionality.
    * @param   {number}  [options.retry.maxRetries=10]               Retry failed requests X times.
    * @param   {object}  [options.headers]                           Additional headers that will be added to the outgoing requests.
+   * @param   {object}  [options.proxy]                             Add the `superagent-proxy` dependency and specify a proxy url eg 'https://myproxy.com:1234'
    * @param   {object}  [options.includeResponseHeaders]            Include the response headers in the payload in the format `{ data, headers }`.
```

Comment:
I think this should be of type string? Also I guess we should fix the `includeResponseHeaders` type too
```suggestion
   * @param   {string}  [options.proxy]                             Add the `superagent-proxy` dependency and specify a proxy url eg 'https://myproxy.com:1234'
   * @param   {boolean}  [options.includeResponseHeaders]            Include the response headers in the payload in the format `{ data, headers }`.
```

Reply:
d'oh - thanks for spotting that!